src/solvers/lowering/byte_operators.cpp

```diff
@@ -182,11 +203,38 @@ static exprt unpack_rec(
         throw non_byte_alignedt(struct_type, comp, *element_width);
       }
 
+      exprt new_offset = nil_exprt();
+      auto new_offset_int = numeric_cast<mp_integer>(offset);
+      exprt new_max_bytes = nil_exprt();
+      auto new_max_bytes_int = numeric_cast<mp_integer>(max_bytes);
+
+      if(new_offset_int.has_value())
+      {
+        *new_offset_int -= member_offset_bits / 8;
+        if(*new_offset_int >= 0)
+          new_offset = from_integer(*new_offset_int, offset.type());
```

Comment:
Comment that the `else` is intentionally missing

Reply:
Done.